tests/test_slow.py

```diff
@@ -57,7 +59,7 @@ def test_simple_simul():
     params.nek.general.write_interval = 500
 
     params.nek.general.variable_dt = False
-    params.nek.general.dt = 0.05
+    params.nek.general.dt = -0.05
```

Comment:
Why negative dt ? What does it mean? Could you please add a comment (in the line before) to explain in few words.

Reply:
It means fixed `dt`, but we don't need it.